footer.php

```diff
@@ -185,17 +172,31 @@ function draw_navbuttons() {
         }
         if ($first) {
             list($icon, $bgcolour) = navbutton_get_icon($settings->buttonstype, 'first', $context, BLOCK_NAVBUTTONS_FIRSTICON, $settings->backgroundcolour, $settings->customusebackground);
-            $output .= make_navbutton($icon, $bgcolour, $first->name, $first->link);
+            $output .= make_navbutton($icon, $bgcolour, $first->name, $first->link, false, "first");
```

Comment:
If you're going to add a new param (and this seems like a good idea to me), please put it before the optional final param (as it doesn't make sense to have to specify 'false' on every call, just to allow you to then specify the class name).